matchbox_socket/src/webrtc_socket/mod.rs

```diff
@@ -42,18 +42,13 @@ type Packet = Box<[u8]>;
 /// See [`WebRtcSocket::new_with_config`]
 #[derive(Debug)]
 pub struct WebRtcSocketConfig {
-    /// The url for the room to connect to
+    /// The url for the session to connect to
     ///
     /// This is a websocket url, starting with `ws://` or `wss://` followed by
-    /// the hostname and path to a matchbox server, followed by a room id and
-    /// optional query parameters.
+    /// the hostname and path to a matchbox server.
```

Comment:
I don't want to remove this information *before* there are good (and documented) alternatives to `matchbox_server` in place

Reply:
Good idea.